package.json

```diff
@@ -49,7 +49,7 @@
     "test": "npm run test:main && npm run test:fp",
     "test:fp": "node test/test-fp",
     "test:main": "node test/test",
-    "validate": "npm run style & npm run test"
+    "validate": "npm run style & npm run test & wait"
   },
```

Comment:
Cool. I'm temp removing it (while I bump to ensure tests pass) and will re-activate it after.